pinot-common/src/test/java/org/apache/pinot/common/data/FieldSpecTest.java

```diff
@@ -268,18 +268,18 @@ public Object[][] provideTestFormatData() {
     entries.add(new Object[]{name, dataType, "-1:HOURS:EPOCH", granularity, true, null});
     entries.add(new Object[]{name, dataType, "0.1:HOURS:EPOCH", granularity, true, null});
     entries.add(new Object[]{
-        name, dataType, "1:HOURS:EPOCH", granularity, false,
-        new DateTimeFieldSpec(name, dataType, "1:HOURS:EPOCH", granularity)
+        name, dataType, "1:HOURS:EPOCH", granularity, false, new DateTimeFieldSpec(name, dataType, "1:HOURS:EPOCH",
```

Comment:
Can you please revert this file? The changes no longer apply

Reply:
Done